e2e/network_test.go

```diff
@@ -15,6 +16,12 @@ import (
 )
 
 func TestNetwork(t *testing.T) {
+	// check if kube-calico daemonset exists
+	// if absent skip this test
+	if _, err := client.ExtensionsV1beta1().DaemonSets("kube-system").Get("kube-calico", metav1.GetOptions{}); err != nil && apierrors.IsNotFound(err) {
```

Comment:
I'm okay with this if we think of the name being stable. If not, is there a label query we could rely on instead?